src/ui/gdrive/gdrive.js

```diff
@@ -27,9 +27,31 @@ goog.require('goog.dom.dataset');
 goog.require('goog.events');
 goog.require('goog.events.EventType');
 goog.require('goog.ui.Dialog');
+goog.require('goog.ui.Dialog.EventType');
 goog.require('goog.ui.Menu');
 goog.require('goog.ui.MenuItem');
 
+var PAGE_SIZE = 1000;
+var ORDER_BY = 'folder,modifiedTime desc,name';
+var FILE_FIELDS = (
+    'files(id,mimeType,parents,iconLink,modifiedTime,name,owners)');
+var FOLDER_MIME_TYPE = 'application/vnd.google-apps.folder';
+var EXT_TO_MIME_TYPE = {
```

Comment:
Please move these "global" definitions to an new file under "src/config/gdrive.js" with the namespace 'cwc.config.GDrive...'.